### ISSUE TITLE
Add warning when you try to pass static params in "execute-command"

### ISSUE THREAD

**Issue by gdubicki** [adnanh/webhook#151]
I tried to pass static params in `"execute-command"` like this:

```
"execute-command": "my_script.py some_static_param"
```

...instead of correctly doing it like this:

```
"execute-command": "my_script.py",
"pass-arguments-to-command": [ "source": "string", "name": "some_static_param" ]
```

It would be nice if webhook could provide tips with common/obvious problems in configuration such as this one.

In this case it could be triggered after detecting whitespace in "execute-command".

**Comment by adnanh** [comment]
I would gladly accept a good PR for this feature :-)

**Comment by adnanh** [comment]
Fixed in #164
